lib/internal/inspector_node_domains.js

```diff
@@ -0,0 +1,433 @@
+'use strict';
+
+const assert = require('assert');
+const EventEmitter = require('events');
+const util = require('util');
+
+const { registerDispatcherFactory } = process.binding('inspector');
+
+const domainHandlerClasses = new Map();
+const SessionTerminatedSymbol = Symbol('DisposeAgent');
```

Comment:
Can the Symbol string match the variable name here, so it’s more obvious what the symbol refers to?

Reply:
Done.